lib/embulk/filter/ruby_proc.rb

```diff
@@ -146,19 +161,31 @@ def self.parse_row_procs(rows, evaluator_binding)
         }.compact
       end
 
+      def self.parse_page_procs(rows, evaluator_binding)
+        rows.map {|rowdef|
```

Comment:
I think that `rows` and `rowdef` are not suiltable name for `parse_page_procs` method.